packages/p2p/src/stores/sendbird-store.js

```diff
@@ -107,7 +107,6 @@ export default class SendbirdStore extends BaseStore {
     }
 
     initialiseChatWsConnection() {
-        this.terminateChatWsConnection();
```

Comment:
There is no need for this call since connection is already closed before initialization.